test/01_function/CMakeLists.txt

```diff
@@ -1,14 +1,12 @@
-include(chimeraBindingTest)
-
-chimera_add_binding_test_boost_python(function_boost_python
+chimera_add_test_boost_python(function_boost_python
```

Comment:
I used `binding_test` instead of just `test` to distinguish binding tests and possible tests for chimera code itself. I understand it wouldn't be so easy to cover all the chimera code but I think we could have some tests. Let me know what do you think.